Dockerfile

```diff
@@ -89,18 +62,16 @@ FROM system as install
 ARG BTYP
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    libsdl1.2debian libsdl-mixer1.2 libsdl-image1.2 libgtk2.0-0 && \
+    libsdl2-2.0 libsdl2-mixer-2.0 libsdl2-image-2.0 libgtk2.0-0 && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-## copy libs first since they are less likely to change
-COPY --from=builder /usr/local/lib /usr/local/lib
-ENV LD_LIBRARY_PATH "${LD_LIBRARY_PATH}:/usr/local/lib"
-
 ## ctp2CD/ copy done in install stage such that stages before are compatible with travis docker build, results in one additional layer in the final DI (incr. DI download size)
 COPY ctp2CD/ /opt/ctp2/
 ## ctp2/ copy has to be after ctp2CD/ to overwrite with newer versions from civctp2
 COPY --from=builder /opt/ctp2/ /opt/ctp2/
+## Disable English layout files
+RUN mv /opt/ctp2/ctp2_data/english/uidata/layouts /opt/ctp2/ctp2_data/english/uidata/layouts.org
```

Comment:
This is something I do not understand. Why is it needed?

Reply:
It is probably temporarily I added this based on #267 which caused crashes during startup of the game. I did not try recently if it is still needed. It moves the original ./ctp2/ctp2_data/english/uidata/layouts out of the way (ctp2 will not be able to read it anymore) without deleting it.

Reply:
Just tested, this causes the GL-CI build to fail because /opt/ctp2/ctp2_data/english/uidata/layouts does not exist there. Please test if it works without these lines for you with current master, i.e. including #288.
You can "amend" this change by doing a `git rebase -i bba27706^' and choosing `e` instead of `pick`, then when rebase stops, just remove the lines and do a `git add -u && git commit --amend` followed by a `git rebase --continue`, then force push.

Reply:
Without these lines it compiles all right on GL-CI and even passes all tests. Sadly, some tests still need multiple tries (like e.g. https://gitlab.com/LynxAbraxas/civctp2/-/jobs/511036310), a flaw I was hoping to vanish as well.

Reply:
Sorry I should not have committed those lines. Still I will check the master first and if still needed keep the change local.

Reply:
Checked with the latest master and ctp2 runs without modifications of the file-/directory-names.